src/cloudWatchLogs/commands/searchLogGroup.ts

```diff
@@ -175,7 +176,7 @@ export class SearchLogGroupWizard extends Wizard<SearchLogGroupWizardResponse> {
         })
 
         this.form.submenuResponse.bindPrompter(createRegionSubmenu)
-        this.form.filterPattern.bindPrompter(createFilterpatternPrompter)
+        this.form.filterPattern.bindPrompter(() => createFilterpatternPrompter(logGroupInfo?.groupName ?? ''))
```

Comment:
```suggestion
        this.form.filterPattern.bindPrompter(({ submenuResponse }) => createFilterpatternPrompter(submenuResponse!.data))
```

You'd want to cover the non-null assertion with a test which I think it already is. I gotta make a PR to explicitly define dependencies to remove these type assertions.